### PR TITLE
Post meta boxes

### PR DESCRIPTION
**Pull request by brenoalvs**
Related to:
https://github.com/brenoalvs/monk/issues/23

### CODE REVIEW

File: admin/class-monk-admin.php [hunk 1]
```diff
@@ -182,11 +182,120 @@ public function monk_active_languages_render() {
...
+							<?php echo esc_html( $lang_name ); ?>
+						</option>
+				<?php   }
+						endforeach; ?>
+				</select>
+			</p>
+		</div>
+
+		<?php
+		} else { ?>
```
Comment (brenoalvs) [inline admin/class-monk-admin.php]: Use the dot syntax `else :`

File: admin/class-monk-admin.php [hunk 2]
```diff
@@ -182,11 +182,70 @@ public function monk_active_languages_render() {
...
+	/**
+	 * Function to create the main language meta box for posts
+	 *
+	 * @since    1.0.0
+	*/
+	public function monk_post_meta_box() {
+		add_meta_box(
+			'monk_post_meta_box_field',
+			__( 'Post languages', 'monk' ),
+			array( $this, 'monk_post_meta_box_field_render'),
```
Comment (brenoalvs) [inline admin/class-monk-admin.php]: Fix brackets code standard

File: admin/class-monk-admin.php [hunk 3]
```diff
@@ -182,11 +182,70 @@ public function monk_active_languages_render() {
...
+		require_once plugin_dir_path( __FILE__ ) . '/partials/admin-monk-post-meta-box-field-render.php';
+	}
+
+	/**
+	 * Function to save data from the monk post meta box
+	 *
+	 * @since    1.0.0
+	*/
+	public function monk_save_post_meta_box( $post_id ) {
+		if ( !isset( $_POST['monk_post_meta_box_nonce'] ) || !wp_verify_nonce( $_POST['monk_post_meta_box_nonce'], basename( __FILE__ ) ) ) {
```
Comment (brenoalvs) [inline admin/class-monk-admin.php]: Fix not code standard

File: admin/class-monk-admin.php [hunk 4]
```diff
@@ -182,11 +182,70 @@ public function monk_active_languages_render() {
...
+			return;
+		}
+		if ( defined( 'DOING_AUTOSAVE' ) && DOING_AUTOSAVE) {
```
Comment (brenoalvs) [inline admin/class-monk-admin.php]: Fix brackets code standard

File: admin/js/monk-admin.js [hunk 5]
```diff
@@ -31,6 +31,7 @@
 
 	$( document ).ready( function() {
 		$( document ).on( 'change', 'select[name="monk_default_language"]', function() {
+			console.log('that');
```
Comment (brenoalvs) [inline admin/js/monk-admin.js]: Remove `console.log`

File: admin/partials/admin-monk-post-meta-box-field-render.php [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+ * Provide the view for the monk_post_meta_box_field_render function
+ *
+ * @link       https://github.com/brenoalvs/monk
+ * @since      1.0.0
+ *
+ * @package    Monk
+ * @subpackage Monk/Admin/Partials
+ */
+
+	wp_nonce_field( basename( __FILE__ ), 'monk_post_meta_box_nonce' );
```
Comment (brenoalvs) [inline admin/partials/admin-monk-post-meta-box-field-render.php]: Move logic to method

File: admin/partials/admin-monk-post-meta-box-field-render.php [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+		<label for="monk-post-add-translation"><?php _e( 'Translations', 'monk' ); ?></label>
+		<a class="edit-post-status hide-if-no-js"><span aria-hidden="true" class="monk-add-translation">Add+</span> <span class="screen-reader-text">Add new translation</span></a>
+	</div>
+	<div class="monk-post-meta-add-translation">
+		<select name="monk_post_add_translation[]">
+			<?php
+				foreach ( $available_languages as $lang_code => $lang_name ) :
+				$lang_id = sanitize_title( $lang_code );
+					if ( in_array( $lang_code, $active_languages ) && $post_default_language != $lang_name ) : ?>
+					<option  value="<?php echo $lang_code ?>" <?php selected( in_array( $lang_code, $post_translations) ); ?> />
```
Comment (brenoalvs) [inline admin/partials/admin-monk-post-meta-box-field-render.php]: Fix brackets code standard

**Review by brenoalvs — COMMENTED** [review]
@chandonn make these fixes

File: admin/class-monk-admin.php (hunk 2 above)
Comment (chandonn-zz) [reply admin/class-monk-admin.php]: Fixed

File: admin/class-monk-admin.php (hunk 3 above)
Comment (chandonn-zz) [reply admin/class-monk-admin.php]: Fixed

File: admin/class-monk-admin.php (hunk 4 above)
Comment (chandonn-zz) [reply admin/class-monk-admin.php]: Fixed

File: admin/class-monk-admin.php (hunk 1 above)
Comment (chandonn-zz) [reply admin/class-monk-admin.php]: Fixed

File: admin/js/monk-admin.js (hunk 5 above)
Comment (chandonn-zz) [reply admin/js/monk-admin.js]: Fixed

File: admin/partials/admin-monk-post-meta-box-field-render.php (hunk 6 above)
Comment (chandonn-zz) [reply admin/partials/admin-monk-post-meta-box-field-render.php]: Fixed

File: admin/partials/admin-monk-post-meta-box-field-render.php (hunk 7 above)
Comment (chandonn-zz) [reply admin/partials/admin-monk-post-meta-box-field-render.php]: Fixed

**Pull request merged**